examples/grpc/src/main/java/io/opentelemetry/example/HelloWorldClient.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2015 The gRPC Authors
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.example;
+
+import io.grpc.*;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.context.propagation.HttpTextFormat;
+import io.opentelemetry.exporters.inmemory.InMemorySpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.SpanData;
+import io.opentelemetry.sdk.trace.TracerSdkFactory;
+import io.opentelemetry.sdk.trace.export.SimpleSpansProcessor;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.SpanContext;
+import io.opentelemetry.trace.Status;
+import io.opentelemetry.trace.Tracer;
+
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class HelloWorldClient {
+  private static final Logger logger = Logger.getLogger(HelloWorldClient.class.getName());
+  private final ManagedChannel channel;
+  private final String serverHostname;
+  private final Integer serverPort;
+  private final GreeterGrpc.GreeterBlockingStub blockingStub;
+
+  // OTel API
+  Tracer tracer;
+  // Export traces in memory
+  InMemorySpanExporter inMemexporter = InMemorySpanExporter.create();
+  // Share context via text headers
+  HttpTextFormat<SpanContext> textFormat;
```

Comment:
In the "http sample" we removed this field in favor of `getHttpTextFormat` access when needed. Could we do this here as well?